src/celeritas/field/detail/FieldUtils.hh

```diff
@@ -16,43 +17,25 @@
 #include "celeritas/Types.hh"
 
 #include "../Types.hh"
+using std::cout;
```

Comment:
Are these used?

Reply:
Whoops I left some debug code in, thanks for the catch!